src/eShop.ServiceDefaults/OpenApi.Extensions.cs

```diff
@@ -26,7 +25,13 @@ public static IApplicationBuilder UseDefaultOpenApi(this WebApplication app)
 
         if (app.Environment.IsDevelopment())
         {
-            app.MapScalarApiReference();
+            app.MapScalarApiReference(options =>
+            {
+                // Empty CDN url to support loading JavaScript assets from NuGet package
+                options.CdnUrl = string.Empty;
```

Comment:
Please don’t ask how I stumbled upon this PR 😄 
Since `CdnUrl` is now `null` by default, setting it to `string.Empty` isn’t necessary.

Reply:
Oooh! I hadn't realized this had changed. Good to know -- avoiding the CDN is nice default behavior.